schemas/restrictions.js

```diff
@@ -0,0 +1,39 @@
+/*!
+ * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+ */
+
+export default {
+  restriction: {title: 'restriction',
+    type: 'object',
+    required: ['zone', 'resource', 'method', 'methodOptions'],
+    additionalProperties: false,
+    properties: {
+      zone: {
+        type: 'string'
+      },
+      resource: {
+        type: 'string'
+      },
+      method: {
+        type: 'string',
+        enum: ['limitOverDuration']
+      },
+      methodOptions: {
+        type: 'object',
+        required: ['limit', 'duration'],
+        additionalProperties: false,
+        properties: {
+          limit: {
+            type: 'integer',
+            minimum: 1
+          },
+          duration: {
+            type: 'string',
+            // eslint-disable-next-line
+            pattern: 'P(?:(\d+(?:[\.,]\d+)?W)|(\d+(?:[\.,]\d+)?Y)?(\d+(?:[\.,]\d+)?M)?(\d+(?:[\.,]\d+)?D)?(?:T(\d+(?:[\.,]\d+)?H)?(\d+(?:[\.,]\d+)?M)?(\d+(?:[\.,]\d+)?S)?)?)'
```

Comment:
TIL that RegExp actually has a `toString` API which allows one to use the export from https://github.com/tolu/ISO8601-duration/blob/master/lib/index.js#L37

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/toString

Reply:
TIL as well! The `/` at the beginning and end had to be replaced, but it worked. Thanks!